GameServer/gameutils/Util.cs

```diff
@@ -411,7 +411,7 @@ public static StackTrace GetThreadStack(Thread thread)
 
 			try
 			{
-				trace = new StackTrace(thread, true);
+				trace = new StackTrace(true);

```

Comment:
This function doesn't work, it should returns the StackTrace from the thread in argument but it's returning the StackTrace of the current thread.

Reply:
`new StackTrace(Thread, bool)` has been deprecated for quite some time without any replacement.

Reply:
If there is no replacement, it's better to throw a NotImplemented (or returning null) than keeping a buggy behavior especially for a debugging tool.

Reply:
Replaced calls with `Util.GetFormattedStackTraceFrom(Thread)` and using Microsoft.Diagnostics.Runtime for StackTrace output.